docs/book/src/developer/tilt.md

```diff
@@ -34,6 +34,11 @@ You can see the status of the cluster with:
 kubectl cluster-info --context kind-capi-test
 ```
 
+### Using a local image registry
+You can configure a local registry by following the instructions available here: https://github.com/tilt-dev/kind-local.
```

Comment:
Maybe mention that the `kind-install-for-capd.sh` script will install a registry too.

Reply:
Thanks! I didn't know that at all. I will try this out and maybe reference the script instead of external resource.

Reply:
I think I'd forgotten :laughing: Was just wondering why I hadn't followed this guide in a long time!

Reply:
Thank you @killianmuldoon! Pr is updated to use the makefile target instead.